packages/catalog-model/src/entity/policies/FieldFormatEntityPolicy.ts

```diff
@@ -161,7 +165,7 @@ export class FieldFormatEntityPolicy implements EntityPolicy {
       optional(
         `links.${i}.icon`,
         links[i]?.icon,
-        KubernetesValidatorFunctions.isValidObjectName,
+        CommonValidatorFunctions.isValidIcon,
```

Comment:
Let's simply use `isNonEmptyString` instead. The icon IDs don't really have a very strict format, and it's a bit overkill to create a bespoke validation function for them.